src/libraries/sendtohelixhelp.proj

```diff
@@ -124,6 +124,11 @@
     <HelixPreCommand Include="printenv | grep COMPlus" />
   </ItemGroup>
 
+  <ItemGroup>
+    <HelixPreCommand Condition="'$(WindowsShell)' == 'true'" Include="set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1" />
+    <HelixPreCommand Condition="'$(WindowsShell)' != 'true'" Include="export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1" />
+  </ItemGroup>
```

Comment:
I think this shouldn't be needed because we set `DOTNET_CLI_HOME` to a writeable location now, its line 132-133.